pcsx2/Recording/Utilities/InputRecordingLogger.h

```diff
@@ -28,32 +28,7 @@
 
 namespace inputRec
 {
-	static void log(const std::string log)
-	{
-		if (log.empty())
-			return;
-
-		recordingConLog(fmt::format("[REC]: {}\n", log));
-
-		// NOTE - Color is not currently used for OSD logs
-		if (GSosdLog)
-			GSosdLog(log.c_str(), wxGetApp().GetProgramLog()->GetRGBA(ConsoleColors::Color_StrongMagenta));
-	}
-
-	static void consoleLog(const std::string log)
-	{
-		if (log.empty())
-			return;
-
-		recordingConLog(fmt::format("[REC]: {}\n", log));
-	}
-
-	static void consoleMultiLog(std::vector<std::string> logs)
-	{
-		std::string log;
-		for (std::string l : logs)
-			log.append(fmt::format("[REC]: {}\n", l));
-
-		recordingConLog(log);
-	}
+	extern void log(const std::string log);
```

Comment:
There's absolutely no need to extern these.

Reply:
Can you explain further?  My understanding is that leaving them static duplicates the functions for every translation unit (everywhere it's included).  And it was recommended to me to switch to `extern` instead.

Reply:
You already put the functions in a different translation unit without declaring them static, that's enough to avoid them being declared in every translation unit. So, functions are "extern" by default when they are not static.

Further declaring them `extern` is completely unnecessary.

Reply:
Ah I see, thanks for the clarification.